src/data/primary-code-paths.ts

```diff
@@ -0,0 +1,67 @@
+export const qiCorePrimaryCodePaths: Record<string, string> = {
```

Comment:
Suggest renaming this file to have QICore and the version in the name. Will make it clearer what it is sourced from and make it easier to add QICore 6.0.0 later.